docs/configuration/index.md

```diff
@@ -2103,16 +2103,18 @@ In addition to the normal cache metrics, the caffeine cache implementation also
-|`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
-|`druid.cache.numConnections`|Number of memcached connections to use.|1|
-|`druid.cache.protocol`|Memcached communication protocol. Can be binary or text.|binary|
-|`druid.cache.locator`|Memcached locator. Can be consistent or array_mod.|consistent|
+| Property                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Default           |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| `druid.cache.expiration`      | Memcached [expiration time](https://code.google.com/p/memcached/wiki/NewCommands#Standard_Protocol).                                                                                                                                                                                                                                                                                                                                                                            | 2592000 (30 days) |
+| `druid.cache.timeout`         | Maximum time in milliseconds to wait for a response from Memcached.                                                                                                                                                                                                                                                                                                                                                                                                             | 500               |
+| `druid.cache.hosts`           | Comma separated list of Memcached hosts `<host:port>`. Needed only when druid.cache.clientMode=static. Dynamic mode [automatically identifies nodes in your cluster](https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/AutoDiscovery.html)                                                                                                                                                                                                                            | none              |
```

Comment:
are we sure this is not needed in dynamic mode? How does one specify which cluster to connect to?